ui/src/testing/stubs/apps-api-stub.ts

```diff
@@ -74,9 +72,7 @@ export class AppsApiStub extends AppsApi {
   public listAppsResponse: ListAppsResponse;
 
   constructor() {
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.